app/views/admin/feedback_messages/_feedback_message.html.erb

```diff
@@ -202,6 +202,47 @@
 </details>
 
 <script>
+
+ document.getElementById('tab-reporter-' + <%= feedback_message.id %>).addEventListener('click', function (event) {
```

Comment:
This entire code change in script change is done so as to get related to tab-related-bootstrap classes by replacing them with correct logic.
In this we activate/inactive tabs and make its content visible/hidden all by using `showTabContent` function.

If there is some better way to optimise this, please do let me know.

Thanks.